### ISSUE TITLE
Calendar flips Date and Month around

### ISSUE THREAD

**Issue by OssiPesonen** [fomantic/Fomantic-UI#986]
# Bug Report

For reasons unknown our web app, which is using Fomantic UI, keeps flipping the month and date around when the input value is set to format 'j.n.Y H:i' and calendar formatter to day.month..year.

**This only happens on Chromium browsers, Chrome, Opera...** Not certain if it is locale specific (I am using Finnish)

For clarification, I am using PHP date() function format. j = day, n = month, Y = year if someone does not understand.

## Steps to reproduce
1. Set input value as php date `j.n.Y H:i`
2. Open view with Google Chrome which then formats the value and calls (at the bottom of the page), $('input').calendar()
3. Input (UI) shows n.j.Y H.i even though dev tools report j.n.Y H:i. This can only happen when date is below 13th making it eligible to pass as a month. When form is submitted the wrong format gets passed forward.

The calendar is formatted as such:

```    
$('#time').calendar({
    type: 'datetime',
    today: true,
    ampm: false,
    firstDayOfWeek: 1,
    text: {
      days: ['S', 'M', 'T', 'K', 'T', 'P', 'L'],
      months: ['Tammikuu', 'Helmikuu', 'Maaliskuu', 'Huhtikuu', 'Toukokuu', 'Kesäkuu', 'Heinäkuu', 'Elokuu', 'Syyskuu', 'Lokakuu', 'Marraskuu', 'Joulukuu'],
      monthsShort: ['Tam', 'Hel', 'Maa', 'Huh', 'Tou', 'Kes', 'Hei', 'Elo', 'Syy', 'Lok', 'Mar', 'Jou'],
      today: 'Tänään',
      now: 'Nyt',
      am: 'AM',
      pm: 'PM'
    },
    monthFirst: false,
    formatter: {
        date: function (date, settings) {
            if (!date) return '';
            var day = date.getDate();
            var month = date.getMonth() + 1;
            var year = date.getFullYear();
            return day + '.' + month + '.' + year;
        }
    }
});
```

The input field (Laravel blade view) is printed as so:

```
<input type="text" placeholder="{{ __("Aika") }}" name="time" value="{{ !empty($obj->time) ? date('j.n.Y H:i', strtotime($obj->time)) : '' }}">
```

## Expected result

Input (UI) field should contain the time format as j.n.Y H:i

## Actual result

Input shows date in format n.j.Y H:i 

## Testcase
https://jsfiddle.net/eow4Lqg5/

Open with Google Chrome or Opera. Date is set to 1st of September 2019 12:00 (01.09.2019)

## Screenshot (when possible)

Input (UI) field:
![image](https://user-images.githubusercontent.com/3592446/64079713-58c5fb80-ccf3-11e9-89b1-95f5224f8310.png)

Dev-tools reported 'value':
![image](https://user-images.githubusercontent.com/3592446/64079720-65e2ea80-ccf3-11e9-9441-620bfe90680c.png)

## Version
2.7.6

**Comment by prudho** [comment]
Please take a look at his PR #995, it may be helpful for your issue.

**Comment by lubber-de** [comment]
As @prudho said, i can confirm it is fixed by #995 
See your adjusted jsfiddle here https://jsfiddle.net/p04xfcnv/
